packages/react/src/@types/assets/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.svg' {
```

Comment:
Let's us import .svg as a component in React (something we were already doing, but needs a little extra help with TS in the mix)